services/system/AuthEcSys/include/services/system/AuthEcSys.hpp

```diff
@@ -7,38 +7,71 @@
 
 namespace SystemService
 {
-   struct AuthRecord
-   {
-      psibase::AccountNumber account;
-      psibase::PublicKey     pubkey;
+   namespace AuthEc
+   {  // Using a namespace here disambiguates this class from the one in AuthSys so docs don't get confused
+      /// A record containing the authorization claims needed for an account using this auth service.
+      struct AuthRecord
+      {
+         /// The account whose transactions will be required to contain the specified public key.
+         psibase::AccountNumber account;
+         
+         /// The public key included in the claims for each transaction sent by this account.
+         psibase::PublicKey     pubkey;
 
-      auto byPubkey() const { return std::tuple{pubkey, account}; }
-   };
-   PSIO_REFLECT(AuthRecord, account, pubkey)
+         auto byPubkey() const { return std::tuple{pubkey, account}; }
+      };
+      PSIO_REFLECT(AuthRecord, account, pubkey)
+   } // namespace AuthEc 
 
+   /// The `auth-ec-sys` service is a default auth service that can be used to authenticate actions for accounts.
```

Comment:
I'm not sure what a "default" auth service is.

Reply:
Reworded